docs/release.rst

```diff
@@ -27,8 +27,11 @@ Major versions are "breaking" in that they can require configuration changes by
 end users.
 
 Before any release, update ``setup.py`` to match the correct version!
-TODO: use https://github.com/warner/python-versioneer or similar.
-
+TODO: use https://github.com/warner/python-versioneer or similar. Also, please
+ensure the integration test suite passes and run `bandit
+<https://bandit.readthedocs.io/en/latest/>`_ being sure to manually inspect any
```

Comment:
Have you run this tool against current master? Should we add it to the dev requirements?

Reply:
Yes and no -- i don't think it's really a dev requirement